src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Safeguarding.cshtml

```diff
@@ -56,9 +56,8 @@
             <p>Use the <a href="https://nhssafeguarding.app/?nocache=0.23461449202295054" id="nhs-app" class="nhs">NHS safeguarding app</a> for guidance on reporting safeguarding concerns.</p>
         </div>
 
-        <form method="post" novalidate data-validate='my-form-validation'>
-            <div class="govuk-form-group">
-
+        <form method="post" novalidate>
+            <div type="submit" class="govuk-form-group">
```

Comment:
I still don't see why we need a form here, but we can revisit it in the next phase.

The `type="submit"` should be on the button rather than the div.

Reply:
Ooops moved submit value into the correct place